src/sst/core/sstinfo.cc

```diff
@@ -152,10 +152,10 @@ addELI(ElemLoader& loader, const std::string& lib, bool optional)
 static void
 processSSTElementFiles()
 {
-    std::vector<bool> EntryProcessedArray;
-    ElemLoader        loader(g_searchPath);
-
-    std::vector<std::string> potentialLibs = loader.getPotentialElements();
+    std::vector<bool>        EntryProcessedArray;
+    ElemLoader               loader(g_searchPath);
+    std::vector<std::string> potentialLibs;
+    loader.getPotentialElements(potentialLibs);
```

Comment:
Why did we make this change? I think the old code is better unless there is a good reason for this.